spec/system/homepage/user_visits_homepage_spec.rb

```diff
@@ -106,24 +128,83 @@
       end
     end
 
-    describe "shop url" do
```

Comment:
The shop url is no longer hardcoded in the sidebar hence we've removed this test.